android/key.properties

```diff
@@ -0,0 +1,4 @@
+storePassword=EmVoncCpAT1j
+keyPassword=EmVoncCpAT1j
```

Comment:
Sensitive???

Reply:
We will be tackling this in regards to issue #68 in the near future